common/src/main/scala/com/gu/zuora/encoding/CustomCodecs.scala

```diff
@@ -69,9 +69,14 @@ trait ModelsCodecs {
   implicit val encodeContribution: Encoder[Contribution] = deriveEncoder
 
   implicit val executionErrorCodec: Codec[ExecutionError] = deriveCodec
+
+  implicit val RequestInfoCodec: Codec[RequestInfo] = deriveCodec
+
   implicit val jsonWrapperDecoder: Decoder[JsonWrapper] = Decoder
-    .forProduct3("state", "error", "encrypted")(JsonWrapper.apply)
-    .or(Decoder.forProduct2("state", "error")((s: String, e: Option[ExecutionError]) => JsonWrapper(s, e, encrypted = false)))
+    .forProduct3("state", "error", "requestInfo")(JsonWrapper.apply)
+    .or(Decoder.forProduct3("state", "error", "encrypted")((s: String, e: Option[ExecutionError], en: Boolean) =>
+      JsonWrapper(s, e, RequestInfo(en, testUser = false, failed = false, Nil))))
```

Comment:
Is this for backwards compatibility - tested here? https://github.com/guardian/support-workers/pull/77/files#diff-19f847463e299e4a039b9badecbc343dR26

Reply:
Yes, it can go once this PR: https://github.com/guardian/support-frontend/pull/390 is deployed